File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/UnitOfMeasure.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class UnitOfMeasure {
    @JsonProperty("Abbreviation")
    public String abbreviation;
    @JsonProperty("Name")
    public String name;
    @JsonProperty("QuantityType")
    public String quantityType;
}

File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/UnitOfMeasure.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class UnitOfMeasure
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("Abbreviation")]
        public required string Abbreviation { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Name")]
        public required string Name { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("QuantityType")]
        public required string QuantityType { get; set; }
    }
}
